ogr/ogrsf_frmts/ili/imdreader.cpp

```diff
@@ -369,6 +396,23 @@ void ImdReader::ReadModel(const char *pszFilename) {
     NodeCountMap oAxisCount;
     NodeVector oArcLineTypes;
 
+    const auto TidLookup = [&oTidLookup](const char* pszKey)
```

Comment:
isn't this identical to the TidLookup function defined above?

Reply:
Ahh i see the distinction -- one is using a member lookup and the other the local captured one. (Renaming the member to have m prefix would help readability here!)

Reply:
> one is using a member lookup and the other the local captured one

The member lookup is in another class than the lambda one.